devsiteHelper.py

```diff
@@ -238,31 +264,48 @@ def getIncludeMD(includeTag, lang='en'):
   return result
 
 
-def getIncludeCode(includeTag, lang='en'):
+def getIncludeCode(include_tag, lang='en'):
   # Returns the contents of an includecode file. If the file is not found,
   # it returns a warning into the doc. Otherwise it returns the file.
   # It also handles start and end regions and can unindent code as requested
-  fileRegEx = re.search(r"content_path=\"(.+?)\"", includeTag)
-  regionRegEx = re.search(r"region_tag=\"(.+?)\"", includeTag)
-  dedentRegEx = re.search(r"adjust_indentation=\"(.+?)\"", includeTag)
-  if fileRegEx is None:
-    msg = 'Error: No <code>content_path</code> specified for ' + includeTag
+  file_regex = re.search(r"content_path=\"(.+?)\"", include_tag)
+  region_regex = re.search(r"region_tag=\"(.+?)\"", include_tag)
+  dedent_regex = re.search(r"adjust_indentation=\"(.+?)\"", include_tag)
+  github_regex = re.search(r"github_path=\"(.+?)\"", include_tag)
+
+  # TODO: support as_downloadable="myproject/tutorial/hello.cc" argument.
```

Comment:
nice to have - can you logging.warn if someone tries to use one of the unsupported attributes, so they don't struggle to understand why it's not working.